benchmarks/baseline-Linux-8.10.7.csv

```diff
@@ -0,0 +1,163 @@
+Name,Mean (ps),2*Stdev (ps),Allocated,Copied,Peak Memory
```

Comment:
This is from: https://github.com/IntersectMBO/ouroboros-network/pull/4717 - https://github.com/IntersectMBO/ouroboros-network/pull/4717/files#diff-22eef62c9d7b36d02f5fd5aaada92702e22d6795c464239cb7b8fe0f26ea1e1cR101 that will run a script that runs the benchmarks and if there's no baseline file to compare to will create one. Subsequent runs will run the benchmarks against the baseline file and check for regressions.

I should disable this actually otherwise we will have a lot of failures in our CI

Reply:
if that's automatic, then ok.  Let's see how it goes.